specs/AppLifecycle/Activation/AppLifecycle Activation.md

```diff
@@ -161,18 +161,25 @@ void RegisterForActivation()
 {
     // Register one or more supported image filetypes, an icon,
     // and zero or more verbs for the File Explorer context menu.
+    string imageFileTypes[3] = { ".jpg", ".png", ".bmp" };
+    string verbs[2] = { "view", "edit" };
     ActivationRegistrationManager::RegisterForFileTypeActivation(
-        new string[]{ ".jpg", ".png", ".bmp" },
-        "MyResources.dll",
-        12,
-        new string[]{ "view", "edit" });
+        imageFileTypes,
+        "MyResources.dll", 0,
```

Comment:
nit: I would pick one style or another here for line-wrapping parameters.  Either one per line, or wrap using 'word' style wrapping at a specific point.

Reply:
Fixed.